src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -419,7 +419,7 @@ def _download_blob(*args, **kwargs):
     for blob_name in source_blobs:
         # remove starting path seperator and normalize
         normalized_blob_name = normalize_blob_file_path(None, blob_name)
-        if normalized_blob_name in blobs_to_download:
+        if normalized_blob_name in blobs_to_download and not overwrite:
```

Comment:
Does `overwrite` also control the blob duplication? Shall we keep the warning in case customer is unaware of multiple blobs existence?